ddtrace/internal/datadog/profiling/dd_wrapper/src/sample.cpp

```diff
@@ -316,6 +318,35 @@ Datadog::Sample::push_class_name(std::string_view class_name)
     return true;
 }
 
+bool
+Datadog::Sample::push_monotonic_ns(int64_t _monotonic_ns)
+{
+    // Monotonic times have their epoch at the system start, so they need an
+    // adjustment to the standard epoch
+    // Just set a static for now and use a lambda to compute the offset once
+    const static auto offset = []() {
+        // Get the current epoch time
+        using namespace std::chrono;
+        auto epoch_ns = duration_cast<nanoseconds>(system_clock::now().time_since_epoch()).count();
+
+        // Get the current monotonic time.  Use clock_gettime directly because the standard underspecifies
+        // which clock is actually used in std::chrono
```

Comment:
Looks like `std::chrono::steady_clock` seems to be using `clock_gettime(CLOCK_MONOTONIC, ..)` under the hood. Can this be used instead? or was there a reason to use clock_gettime instead of std::chrono one?